cfgov/v1/jinja2/v1/_includes/organisms/email-signup-form.html

```diff
@@ -0,0 +1,39 @@
+{# ==========================================================================
+
+   email-subscribe-form.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates an email subscription form when given:
+
+   gd_code:       A GovDelivery code for a specified mailing list.
+
+   settings:      An object of used to customize the markup.
+
+   settings.text: The text used used within the description markup.
+
+   ========================================================================== #}
+
+{% macro render(settings, gd_code='') %}
+{% import 'molecules/form-field-with-button.html' as form_field_with_button %}
+<form class="o-email-subscribe-form"
+      id="email-subscribe-form"
```

Comment:
@Scotchester, just ended up using `{{ settings.field.name ~ '_' ~ range(1, 100) | random }}`, which will make it highly probably that it's unique but won't ensure it. 

fixed in https://github.com/cfpb/cfgov-refresh/commit/33ce7fbf735d7e6a4c927e3d806ee44ef1e8ef4d